pkg/app/formatter/table/version.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package table
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/tensorchord/envd/pkg/app/formatter"
+	"github.com/tensorchord/envd/pkg/version"
+)
+
+func PrintVersion(clicontext *cli.Context) error {
+	short := clicontext.Bool("short")
+	detail := clicontext.Bool("detail")
+	ver := version.GetVersion()
+	detailVer, err := formatter.GetDetailedVersion(clicontext)
+	fmt.Printf("envd: %s\n", ver)
+	if short {
+		return nil
+	}
+	fmt.Printf("  BuildDate: %s\n", ver.BuildDate)
```

Comment:
You can make the indentation a const to avoid manually writing it everywhere.

Reply:
SGTM